nilearn/datasets/struct.py

```diff
@@ -9,24 +9,27 @@
 from scipy import ndimage
 from sklearn.utils import Bunch
 
-from .utils import (_get_dataset_dir, _fetch_files,
-                    _get_dataset_descr, _uncompress_file)
+from .utils import (_get_dataset_dir, _fetch_files, _get_dataset_descr)
 
 from .._utils import check_niimg, niimg
-from ..image import new_img_like, get_data
+from ..image import new_img_like, get_data, resampling
+
 
 _package_directory = os.path.dirname(os.path.abspath(__file__))
-# Useful for the very simple examples
-MNI152_FILE_PATH = os.path.join(_package_directory, "data",
-                                "avg152T1_brain.nii.gz")
+MNI152_FILE_PATH = os.path.join(
+    _package_directory, "data",
+    "mni_icbm152_t1_tal_nlin_sym_09a_noskull_rescaled.nii.gz")
+GM_MNI152_FILE_PATH = os.path.join(_package_directory, "data",
+                                   "mni_icbm152_gm_tal_nlin_sym_09a.nii.gz")
+WM_MNI152_FILE_PATH = os.path.join(_package_directory, "data",
+                                   "mni_icbm152_wm_tal_nlin_sym_09a.nii.gz")
 FSAVERAGE5_PATH = os.path.join(_package_directory, "data", "fsaverage5")
 
 
 def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     """Download and load the ICBM152 template (dated 2009).
 
-    For more information, see :footcite:`FONOV2011313`,
-    :footcite:`Fonov2009`, and :footcite:`Collins1999algorithm`.
+    For more information, see [1]_, [2]_, and [3]_.
```

Comment:
These references were improved recently to rely on sphinx-contrib-bibtex. We shouldn't revert them back here.